content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -31,7 +32,8 @@ your specific needs. The following built-in configuration profiles are currently
 
 1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.
 
-1. **remote**: used for configuring remote clusters of a
+1. **remote**: used for configuring
```

Comment:
Consider changing the order, probably have remote before demo profile.

Reply:
Changing the order is beyond what's needed to introduce the terminology is also outside the scope of this PR.